### PR TITLE
css-two-column-layout | Samuel Cho

### PR DESCRIPTION
**Pull request by SamuelCho1**
Css two column layout
Screen Shot 2021-01-22 at 5 45 35 PM

### CODE REVIEW

File: css-two-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  width: 100%;
+}
+
+.header {
+  text-transform: uppercase;
+  letter-spacing: 0.2rem;
+}
+
+.header-text {
+  padding: 0 110px;
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:23]: Is this to center the text? You should use `text-align: center`, avoid large padding and margin values.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Good work so far!! Be careful of large padding and margin values 👀

File: css-two-column-layout/styles.css (hunk 1 above)
Comment (SamuelCho1) [reply css-two-column-layout/styles.css:23]: It was to make the text under the header to not take up the whole width and be on two lines.

**Comment by SamuelCho1** [comment]
Screen Shot 2021-01-23 at 11 22 42 AM

**Review by codingcodymiller — APPROVED** [review]
Nice work!!

**Pull request merged**